BUILD_macos.md

```diff
@@ -70,6 +71,10 @@ PRs with corrections and updates are welcome!
   brew install libfido2
   ```
 
+* To install `yarn` for building the UI
+  * `brew install node yarn`
+  * Currently, Yarn classic (< 2.0) is required, but this is what Homebrew is currently shipping at the time of writing, so it Just Works.
```

Comment:
```suggestion
  * Currently, [`yarn`](https://classic.yarnpkg.com/en/docs/install) (< 2.0.0) is required
```

I don't think there's a need to mention about brew only shipping v1. I can't imagine they'll change it any time soon (the release of yarn 2 wasn't the smoothest, and even `npm install yarn` defaults to v1 still), and if it is changed we can update these docs.